tools/reviewer/web/src/app/dashboard/reviews-panel/review/review.component.html

```diff
@@ -1,31 +1,34 @@
-<div *ngIf="diff" class="single-review">
+<!-- This html file is too big -->
+<!-- TODO: do something with that -->
```

Comment:
Let's combine the 2 lines:
`!-- TODO: Make this html file smaller, or split into separate files -->`

Reply:
I changed a lot in this file, actually most of it has been changed. It got merged a couple days ago.

Reply:
@MateMahinKhan what do you mean? The diff based on fresh state of master.
There's still the huge html file:
https://github.com/google/startup-os/blob/master/tools/reviewer/web/src/app/dashboard/reviews-panel/review/review.component.html

Reply:
@webdevelopland  hmm. so that means I still have these changes. merging will be a tough one for me :D